community/tools/ra-data-hasura/src/index.js

```diff
@@ -32,6 +30,32 @@ export default (serverEndpoint, headers) => {
       throw new Error(JSON.stringify({'error': 'Invalid table/schema resource'}));
     }
 
+    return {schema: schema, tableName: tableName};
+
+  };
+
+  const getPrimaryKey = (resource) => {
+
+    let primaryKey = 'id';
+
+    if (config && config['primaryKey'][resource]) {
+      primaryKey = config['primaryKey'][resource];
+    }
+    return primaryKey;
+  };
+
+  const convertDataRequestToHTTP = (type, resource, params) => {
+    const options = {};
+    let finalQuery = {};
+
+    const tableSchema = getTableSchema(resource);
+
+    let schema = tableSchema.schema;
```

Comment:
You can combine L53-L55 into a single using destructuring:

```javascript
let { schema, tableName } = tableSchema;
```

Reply:
Makes sense 👍